.github/scripts/generate_ci_workflows.py

```diff
@@ -162,50 +166,38 @@ class CIWorkflow:
     is_scheduled: str = ''
     is_default: bool = False
     num_test_shards: int = 1
-    only_run_smoke_tests_on_pull_request: bool = False
-    num_test_shards_on_pull_request: int = -1
-    distributed_test: bool = True
     timeout_after: int = 240
     xcode_version: str = ''
     only_on_pr: bool = False
     ios_arch: str = ''
     ios_platform: str = ''
+    test_jobs: Any = field(default_factory=list)
 
-    # The following variables will be set as environment variables,
-    # so it's easier for both shell and Python scripts to consume it if false is represented as the empty string.
-    enable_jit_legacy_test: YamlShellBool = "''"
-    enable_distributed_test: YamlShellBool = "''"
-    enable_multigpu_test: YamlShellBool = "''"
-    enable_nogpu_no_avx_test: YamlShellBool = "''"
-    enable_nogpu_no_avx2_test: YamlShellBool = "''"
-    enable_slow_test: YamlShellBool = "''"
-    enable_docs_test: YamlShellBool = "''"
-    enable_backwards_compat_test: YamlShellBool = "''"
-    enable_xla_test: YamlShellBool = "''"
-    enable_noarch_test: YamlShellBool = "''"
-    enable_force_on_cpu_test: YamlShellBool = "''"
+    enable_default_test: bool = True
```

Comment:
We no longer need explicit type annotation here, do we?

Reply:
I have no idea lol, the rules for `.github` are different and stricter than other folders, I can try to remove